src/app/workspace/cmdinput/textareainput.tsx

```diff
@@ -339,30 +340,36 @@ class TextAreaInput extends React.Component<{ screen: Screen; onHeightChange: ()
     @boundMethod
     onHistoryKeyDown(e: any) {
         let inputModel = GlobalModel.inputModel;
-        if (e.code == "Escape") {
+        if (GlobalModel.checkKeyPressed(e, "Escape")) {
             e.preventDefault();
             inputModel.resetHistory();
             return;
         }
-        if (e.code == "Enter") {
+        if (GlobalModel.checkKeyPressed(e, "Enter")) {
             e.preventDefault();
             inputModel.grabSelectedHistoryItem();
             return;
         }
-        if (e.code == "KeyG" && e.getModifierState("Control")) {
+        if (GlobalModel.checkKeyPressed(e, GlobalModel.KeyPressModifierControl + ":g")) {
             e.preventDefault();
             inputModel.resetInput();
             return;
         }
-        if (e.code == "KeyC" && e.getModifierState("Control")) {
+        if (GlobalModel.checkKeyPressed(e, GlobalModel.KeyPressModifierControl + ":c")) {
             e.preventDefault();
             inputModel.resetInput();
             return;
         }
         if (
-            e.code == "KeyR" &&
-            (e.getModifierState("Meta") || e.getModifierState("Control")) &&
```

Comment:
some or key bindings here
I feel like we should make these either Ctrl or Cmd, rather than both

Reply:
I think it should just be control?
Looking at this though I understand why or is used here since it's a really important command